data/avro-schema/src/main/resources/avro/net/corda/data/persistence/EntityRequest.avsc

```diff
@@ -12,6 +12,11 @@
       },
       "doc": "Time ({@link java.time.Instant}) in milliseconds when the record was created."
     },
+    {
+      "name": "flowId",
+      "type": "string",
+      "doc": "The unique ID for tracking the flow. Responses will be returned on this key."
+    },
     {
```

Comment:
If we have the `flowId` here then I doubt we also need the `FlowKey`.

Reply:
We need holding id to get the right sandbox. But we don't need the client request id so FlowKey might be overboard.